packages/go_router/lib/src/matching.dart

```diff
@@ -134,6 +134,11 @@ class RouteMatchList {
 
   /// Returns the error that this match intends to display.
   Exception? get error => matches.first.error;
+
+  @override
+  String toString() {
+    return fullpath;
```

Comment:
Can you use the fomart similar to this?
'${objectRuntimeType(this, 'RouteMatchList')}($fullpath)';

This way it will be easier to differentiate the object type when print it. Otherwise it is hard to tell whether it is printing a string  or a RouteMatchList.

Reply:
I did it like this to replicate the behaviour of version 3.0.7. But sure, it makes sense to be more explicit in a `toString`